src/api/index.ts

```diff
@@ -134,6 +136,20 @@ function createDexPriceEstimatorApi(): DexPriceEstimatorApi {
   return dexPriceEstimatorApi
 }
 
+function createTcrApi(web3: Web3): TcrApi {
+  // TODO: load config from config file
+  const config = {
+    1: { listId: 1, contractAddress: '0x1854dae560abb0f399d8badca456663ca5c309d0' },
+    4: { contractAddress: '0xBb840456546496E7640DC09ba9fE06E67C157E1b' },
+  }
+
+  const tcrApi = new TcrApiProxy({ web3, config })
+
+  window['tcrApi'] = tcrApi
```

Comment:
should all these window assignments only be for `development`? or is prod ok

Reply:
I guess so? We could revisit this, but all apis are already that way.
Besides, 'hidden' feature. Is there any harm or extra bundle size for exposing this?